bedrock/products/templates/products/vpn/landing.html

```diff
@@ -55,16 +58,40 @@
   {% endif %}
 
   {% block hero %}
+    {% set desc = None if cyber_security_offer else ftl('vpn-landing-hero-desc') %}
     {% call vpn_hero(
       heading=ftl('vpn-shared-product-name'),
       subheading=ftl('vpn-landing-hero-sub-heading-v2', fallback='vpn-landing-hero-sub-heading'),
-      desc=ftl('vpn-landing-hero-desc')
+      desc=desc
     ) %}
       <div class="js-vpn-scroll-to-pricing">
         {% block hero_cta_change %}
-          <a class="mzp-c-button mzp-t-xl" href="#pricing" data-cta-type="button" data-cta-text="Scroll to pricing" data-cta-position="primary">
-          {{ ftl('vpn-shared-subscribe-link') }}
-          </a>
+          {% if cyber_security_offer %}
+            <p><strong class="t-mark">Limited-time offer</strong> for Cyber Security month:</p>
+            <h2 class="c-limited-offer-heading">
+              <span class="c-limited-offer-price">{{ settings.VPN_LIMITED_OFFER_MONTHLY_PRICE_USD }}</span>/month (save $59.88)
```

Comment:
Assuming copy is still a little up in the air? It seems odd to have the save $59.88 without context (when used in the 12-month plan box, you know it's an annual savings amount)

Reply:
I agree, I'll see what our copywriter thinks. Thanks for the suggestion